pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -166,14 +166,44 @@ public void executionModelAction() throws Exception {
         ExecutionModelAction action = b.getAction(ExecutionModelAction.class);
         assertNotNull(action);
         ModelASTStages stages = action.getStages();
-        assertNull(stages.getSourceLocation());
+        assertExecutionModelActionStageContents(b, stages);
+    }
+
+    @Test
+    public void executionModelActionFullPipeline() throws Exception {
```

Comment:
Essentially the same as the `executionModelAction` test, but uses `ExecutionModelAction.getPipelineDef` and checks for some info that isn't available if you use `ExecutionModelAction.getStages`. I thought it was useful to still keep the old test around to help maintain compatibility for any existing callers.